mkdocs.yml

```diff
@@ -7,20 +7,33 @@ theme:
   hljs_languages:
     - yaml
 nav:
-  - 'index.md'
-  - 'overview.md'
-  - Preparation: 'preparation/index.md'
-  - 'Information Gathering': 'information-gathering/index.md'
-  - 'OpenShift Enhancement': 'openshift-enhancement/index.md'
-  - 'RHCOS': 'rhcos/index.md'
-  - 'Installer': 'installer/index.md'
-  - 'Cloud Controller Manager': 'cloud-controller-manager/index.md'
-  - 'Container Storage Interface Driver': 'container-storage-interface-driver/index.md'
-  - 'Machine API Controllers': 'machine-api-controllers/index.md'
-  - 'Network Ingress and DNS': 'network-ingress-dns/index.md'
-  - 'Infrastructure Metadata Service': 'infrastructure-metadata-service/index.md'
-  - 'Continuous Integration': 'continuous-integration-and-testing/index.md'
-  - 'Component Infrastructure': 'component-infrastructure/index.md'
-  - 'Documentation': 'documentation/index.md'
-  - 'Red Hat Relationship': 'red-hat-relationship/index.md'
-  - 'Release': 'release/index.md'
+  - Welcome: index.md
+  - Full integration:
```

Comment:
this change is aligned with the Platform External documentation, aligned in the draft docs https://github.com/openshift/infrastructure-provider-onboarding-guide/pull/16